src/modules/mavlink/mavlink_parameters.cpp

```diff
@@ -245,6 +245,12 @@ MavlinkParametersManager::handle_message(const mavlink_message_t *msg)
 
 				/* Copy values from msg to uorb using the parameter_rc_channel_index as index */
 				size_t i = map_rc.parameter_rc_channel_index;
+
+				if (i >= sizeof(_rc_param_map.param_index) / sizeof(_rc_param_map.param_index[0])) {
+					PX4_ERR("parameter_rc_channel_index out of bounds");
```

Comment:
Mavlink error so the user knows why it's silently failing?

Reply:
Fixed and force pushed.